python/pyproject.toml

```diff
@@ -13,7 +13,7 @@ classifiers = [
     "Programming Language :: Python :: 3",
     "License :: OSI Approved :: Apache Software License",
 ]
-dependencies = ["requests", "tqdm", "numpy", "IPython", "setproctitle"]
+dependencies = ["requests", "tqdm", "numpy", "IPython", "setproctitle", "turbomind"]
```

Comment:
Do not list it as a dependency. Only install this when people need this quantization.